.env

```diff
@@ -0,0 +1,11 @@
+##
```

Comment:
I'd like to update this so that when we make changes, git isn't showing uncommitted changes - Would hate to have someone with write access to this accidentally commit credentials. Maybe distribute an example `.env` file and add the actual `.env` file to `.gitignore` ? Open to other approaches, as long as the result makes it harder to accidentally commit actual credentials.

Reply:
@christianc1 Actually, this ends up with incorrectly formatted values in the `.wpsnapshots.json` file. Why not just have a .wpsnapshots.json file instead of the env file? its just about as easy to edit, and we can just mount a .wpsnapshots.json file right on top of where we'd expect it in the container `~/.wpsnapshots.json` and we get to bypass the whole config step.